locale/foreman.pot

```diff
@@ -8065,7 +8065,9 @@ msgid "Append domain names to the host"
 msgstr ""
 
 #: ../app/models/setting/general.rb:26
-msgid "Duration in minutes after servers are classed as out of sync."
+msgid ""
```

Comment:
next time, this is done during the release process, so there is no need doing this manually in every PR, but doesn't hurt :)